code/obj/storage/gang_crate.dm

```diff
@@ -0,0 +1,1067 @@
+		lootObject.pixel_x = I.position_x + off_x
+		lootObject.pixel_y = I.position_y + off_y
+		lootObject.layer = I.layer + layer_offset
+		lootObject.AddComponent(/datum/component/transform_on_pickup)
+		return lootObject
+
+	// spawn loot and disappear
+	proc/handle_loot(loc,datum/loot_spawner_info/I)
+		spawn_loot(loc,I)
+		qdel(src)
+
+	// this is what your loot instances should override, to do their thing.
+	proc/spawn_loot(loc,datum/loot_spawner_info/I)
+
+	short //1x1
+		xSize = 1
+		ySize = 1
+		// GANG_CRATE_GUN:
+		makarov
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/gun/kinetic/makarov,scale_x=0.725,scale_y=0.725)
+		microphaser
+			weight = 1
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/gun/energy/phaser_small,rot=90,scale_x=0.8,scale_y=0.725)
+
+		small_nades
+			weight=2
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/old_grenade/stinger/frag,off_y=2,scale_x=0.7,scale_y=0.7)
+				spawn_item(C,I,/obj/item/old_grenade/stinger/frag,off_y=-2,scale_x=0.7,scale_y=0.7)
+		small_stingers
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/old_grenade/stinger,off_y=2,scale_x=0.7,scale_y=0.7)
+				spawn_item(C,I,/obj/item/old_grenade/stinger,off_y=-2,scale_x=0.7,scale_y=0.7)
+
+		// GANG_CRATE_GEAR
+		spraypaint
+			tier = GANG_CRATE_GEAR
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/spray_paint,scale_x=0.7,scale_y=0.6)
+		flash
+			weight = 2
+			tier = GANG_CRATE_GEAR
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/device/flash,off_x=2,off_y=0,rot=0,scale_x=0.6,scale_y=0.6)
+				spawn_item(C,I,/obj/item/device/flash,off_x=-2,off_y=0,rot=0,scale_x=0.6,scale_y=0.6)
+		flashbang
+			weight = 4
+			tier = GANG_CRATE_GEAR
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/chem_grenade/flashbang,off_y=2,scale_y=0.8)
+				spawn_item(C,I,/obj/item/chem_grenade/flashbang,off_y=-2,scale_y=0.8)
+		wiretap
+			weight = 2
+			tier = GANG_CRATE_GEAR
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/device/radio_upgrade)
+		poison_loose
+			weight=1
+			tier = GANG_CRATE_GEAR
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/reagent_containers/glass/bottle/poison)
+
+		//GIMMICKS
+		recharge_cell //maybe good to bribe sec
+			weight=2
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/ammo/power_cell/self_charging/medium,off_y=2)
+		jaffacakes
+			tier = GANG_CRATE_GIMMICK
+			weight=5
+			spawn_loot(var/C,var/I)
+				for(var/i=1 to 4)
+					var/obj/item/cake = spawn_item(C,I,/obj/item/reagent_containers/food/snacks/cookie/jaffa,off_y=2*(2-i),scale_x=0.85,scale_y=0.85)
+					cake.reagents.add_reagent("omnizine", 10)
+					cake.reagents.add_reagent("msg", 1) //make em taste different
+		weed
+			weight=5
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/plant/herb/cannabis/spawnable,off_y=2,scale_x = 0.8,scale_y = 0.8)
```

Comment:
you could probably use `bound_x` and `bound_y` to automate this scaling, but consider that this is a pixel game and scaling things down by fractions tends to look Very Bad so it may not be a good idea in general

Reply:
yeah i've hand-scaled these due to the Not Good scaling